ballerina-tests/tests/configuration_request_limit_config_test.bal

```diff
@@ -109,7 +109,7 @@ service /requestPayloadLimit on lowPayloadLimitEP {
 function testValidUrlLength() {
     http:Client limitClient = checkpanic new("http://localhost:" + requestLimitsTestPort1.toString());
```

Comment:
Lets remove `checkpanic` usage and use `check` instead. Check for other places as well.
```suggestion
function testValidUrlLength() returns error? {
    http:Client limitClient = check new("http://localhost:" + requestLimitsTestPort1.toString());
```